dbos-config.schema.json

```diff
@@ -145,6 +145,9 @@
         }
       }
     },
+    "skip_missed_scheduled_workflows": {
```

Comment:
Maybe add a new section for this instead of it being a standalone?

Reply:
What would you like the section to be?  scheduled_workflows ->  skip_missed ?

Reply:
On second thought, I don't like having this sort of thing in the config. Maybe make it an argument to the decorator itself.  What do you think?

Reply:
I've left room for such a thing in decorator already, so could add there (also?).
I thought we went with the config file so you could use it in dev mode only... though I am not sure it's safe to expect the user to remember to change it back before deploy.

Reply:
I think the decorator approach is better because this config flag is global. People might want different scheduled workflows behave differently.

Reply:
Yeah, agree, it seems risky in the config file

Reply:
Are we doing both or getting rid of config?

Reply:
And which is default?

Reply:
I'd vote for getting rid of config. I like the new `ExactlyOncePerIntervalWhenActive` mode in the decorator -- it's close to the workflow code and easy to understand.

I think the default one should be `ExactlyOncePerIntervalWhenActive` because it's how other programs written with node-cron typically work.